src/vs/workbench/api/common/extHostNotebook.ts

```diff
@@ -434,6 +434,12 @@ export class ExtHostNotebookController implements ExtHostNotebookShape {
 							}
 							finalMatchedTargets.add(uri);
 						});
+					}).catch(_err => {
+						// temporary fix for https://github.com/microsoft/vscode/issues/205044: don't show notebook results for remotehub repos.
```

Comment:
it would be good to log a warning at least since this is swallowing and ignoring _all_ exceptions. Or just ignore the specific exception?

Reply:
Good point, I added a check and a warning log.